docs/features/delayed_creation/README.md

```diff
@@ -0,0 +1,33 @@
+---
+category: api-usage
+order: 60
+url: features/delayed_creation
+menu-title: Delayed creation
+meta-title-shor: Delayed creation
+---
+<!--
+Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Delayed creation
+
+<info-box info="">This feature was introduced in 4.17.0 and by default it is off. It requires [enabling config option](../api/CKEDITOR_config.html#cfg-delayIfDetached) to worksfor the editor.</info-box>
+
+It was introduced because there were some issues during editor creation on the element that is detached from the DOM. During the entire process, there is some information extracted from the editor elements document. If the target element is detached, it is simply impossible to access them. This situation may be caused unintentionally by using CKEditor 4 with popular frameworks (e.g. [Angular integration](../guide/dev_angular.html)) which may detach elements that are hidden.
```

Comment:
I would start by saying how it works / what it does. So that "it prevents editor creation when parent element is detached from DOM...".

Then maybe a separate section on possible issue it can solve (for example using CKEditor 4 in modals, using with integrations and so on). So the similar description you had already added. This can give broader context and clarity when to use it.

Reply:
One more thing, apart from using "some" like:

> there were some issues during editor

or

> During the entire process, there is some information extracted from

You may use more specific examples, or even link to GH issues. So it would be better to describe specific cases more precisely or use term like "specific issues", "group of issues", e.g.:

> It was introduced because we have observed a group of issues when editor initialization failed during editor creation on the element that is detached from the DOM.

Reply:
> You may use more specific examples, or even link to GH issues.

I had doubts about linking to GH issues in documentation. But if you say we can do this... 👍

Reply:
And finally I decide to manage that inside the doc article ;)